### DISCUSSION TITLE
Wish-list: Visualization tools

### DISCUSSION THREAD

**Discussion by arokem** [yeatmanlab/pyAFQ#941] [Ideas]
We would like to build out more automated tools for visualizations of pyAFQ results. This discussion thread is a place to brainstorm, and paste in examples of the kinds of visualizations we would like to have implemented.

**Comment by mayayab** [comment]
I liked being able to plot a statistical value along the tract profile (in matlab, I would get a vector of r values or t values or p values which was saved in the TractProfile of the AFQ structure, and plotted along the tract). This is the function that created those figures: 
https://github.com/yeatmanlab/AFQ/blob/master/3Dmesh/AFQ_RenderFibers.m

![sub40_LeftIFOF_MD_root_profile_rad10](https://user-images.githubusercontent.com/7735190/210473203-7b7f3281-4dad-44af-9e76-77bac88082ca.png)
![sub51_LeftAF_FR_root_profile_rad10](https://user-images.githubusercontent.com/7735190/210473332-eaa8e30e-fc05-4852-a4b1-2e4ca9331398.png)

**Reply by arokem** [reply]
Does this example have the functionality you are looking for? http://yeatmanlab.github.io/pyAFQ/auto_examples/plot_viz.html#adding-core-bundles-with-tract-profiles. And, if not, how could it be improved to give you what you need?
